clientcontroller/interface.go

```diff
@@ -49,6 +50,9 @@ type ClientController interface {
 	// QueryFinalityProviderSlashedOrJailed queries if the finality provider is slashed or jailed
 	QueryFinalityProviderSlashedOrJailed(fpPk *btcec.PublicKey) (slashed bool, jailed bool, err error)
 
+	// EditFinalityProviderDescription edits description of the finality provider
+	EditFinalityProviderDescription(fpPk *btcec.PublicKey, reqDesc sttypes.Description) (*sttypes.Description, error)
```

Comment:
It would be more general if we could have our own description type as our consumer chains might not be a Cosmos chain